Source/ProgressionSystemRuntime/Public/Components/PSSpotComponent.h

```diff
@@ -32,6 +32,10 @@ class PROGRESSIONSYSTEMRUNTIME_API UPSSpotComponent : public UActorComponent
 	UFUNCTION(BlueprintCallable, Category= "C++")
 	void RefreshAmountOfUnlockedSkins(bool bApplySkin);
 
+	/** Returns true if this is a current spot */
+	UFUNCTION(BlueprintCallable, Category= "C++")
```

Comment:
1. `IsCurrentSpot` has to be `const`-function at the end, the same as any other Getter-type function. So it can be called from `const` objects. It's very important point in any coding standard

2. Please add `BlueprintPure` for Get\Is function, so it will not have execution pins in blueprints:
```
	UFUNCTION(BlueprintCallable, BlueprintPure, Category= "C++")
```

Reply:
added https://github.com/moinm3uw/ProgressionSystem/pull/63/commits/abf15f7dbfa3ac25cc51aadcf325b96650023807